### DISCUSSION TITLE
問一下

### DISCUSSION THREAD

**Discussion by jerrylam901** [jynew/jynew#953] [General]
我是用m1 的mac 電腦 
我試git clone 了 jynew 後 安裝好 unity 但並不能下載到 2020.03.25 的配套
如用 2020.03.35 又不能正常運行和打包 
有沒有那位能夠給一個現時最新能在 mac 運行的遊戲分享一下嗎 ?

**Comment by chenggongchina** [comment]
不能下载到配套是什么意思？没看明白，能否附截图？
